AudioBar/AudioBarViewController.swift

```diff
@@ -8,6 +8,8 @@ public final class AudioBarViewController: UIViewController, StoreDelegate {
     private lazy var store: Store<AudioBar> = AudioBar.makeStore(delegate: self, seed: .init())
     private let player = AVPlayer()
     private let volumeView = MPVolumeView()
+    private let commandCenter = MPRemoteCommandCenter.shared()
```

Comment:
Calling this just a _Command Center_ is ambiguous.  Consider `remoteCommandCenter` at least.